client/src/components/gamedashboard/global/cards/CardInvestment.vue

```diff
@@ -2,7 +2,7 @@
   <div class="card-investment v-step-15">
     <div class="ci-container" :style="opacity">
       <div class="type">
-        <p class="name">{{ name }}</p>
+        <p class="name">{{ returnName }}</p>
```

Comment:
minor nitpick, I think this should probably be renamed to `label`. `returnName` isn't a super helpful variable name :sweat_smile: